packages/components/pie-textarea/src/index.ts

```diff
@@ -34,6 +34,9 @@ export class PieTextarea extends FormControlMixin(RtlMixin(LitElement)) implemen
     @property({ type: String })
     public value = defaultProps.value;
 
+    @property({ type: String })
+    public defaultValue?: TextareaProps['defaultValue'];
```

Comment:
Nit: the `?` already comes through from `defs.ts`
```suggestion
    public defaultValue: TextareaProps['defaultValue'];
```